charts/templates/server-repository/configmap.yaml

```diff
@@ -20,7 +20,7 @@ data:
   TESTING: {{ .Values.serverRepository.config.TESTING | quote }}
   NODE_ENV: {{ .Values.serverRepository.config.NODE_ENV | default "development" }}
   UI_DOMAIN_NAME: {{ .Values.serverRepository.config.UI_DOMAIN_NAME }}
```

Comment:
Can we delete `TESTING`, `UI_DOMAIN_NAME` and `REPOSITORY_PATH`?

Can we change the default of `NODE_ENV` to `production`?